modules/clickhouse/src/main/java/org/testcontainers/containers/ClickHouseContainer.java

```diff
@@ -54,8 +56,8 @@ public ClickHouseContainer(final DockerImageName dockerImageName) {
     }
 
     @Override
-    protected Integer getLivenessCheckPort() {
-        return getMappedPort(HTTP_PORT);
+    public Set<Integer> getLivenessCheckPortNumbers() {
+        return Sets.newHashSet(HTTP_PORT);
```

Comment:
just curious - is it to remove the deprecated usage, or it somehow got broken?
Ideally, I would prefer to keep the old code as much as possible, to avoid regressions (and maybe do another pass of removing deprecated usage)

Reply:
As discussed, I've confirmed that this change is not a sign of a breaking change in the PR - i.e. tests pass in these modules with or without this particular change.